python-sdk/src/astro/sql/operators/base_operator.py

```diff
@@ -1,8 +1,9 @@
 from abc import ABC
 
 from airflow.models.baseoperator import BaseOperator
+from airflow.utils.log.logging_mixin import LoggingMixin
 from astro.sql.operators.upstream_task_mixin import UpstreamTaskMixin
 
 
-class AstroSQLBaseOperator(UpstreamTaskMixin, BaseOperator, ABC):
+class AstroSQLBaseOperator(UpstreamTaskMixin, BaseOperator, LoggingMixin, ABC):
```

Comment:
The `BaseOperator` inherits `LoggingMixin` via `AbstractOperator` class so this is not needed

Reply:
Try upgrading `apache-airflow` in the environment that your IDE use -- nevertheless, it is something in your local setup

https://github.com/apache/airflow/blob/2.4.1/airflow/models/baseoperator.py#L438-L439
https://github.com/apache/airflow/blob/2.4.1/airflow/models/abstractoperator.py#L67